.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions" # See documentation for possible values
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "weekly"
+    groups:
+      dependencies:
```

Comment:
`gha` or something that's obvious this is updating the GitHub Actions.

Reply:
Line 3?

Reply:
Or do you mean for the commit-message or prefix?

https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#commit-message

Reply:
I mean this line, which is the name of the group (and used for commit too).